apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -108,7 +108,7 @@ function WpcomNux() {
 				<DocumentationImage className="edit-post-welcome-guide__image" />
 				<p className="edit-post-welcome-guide__text">
 					{ __( 'New to the WordPress editor? Want to learn more about using it? ' ) }
-					<ExternalLink href={ __( 'https://support.wordpress.com/wordpress-editor/' ) }>
+					<ExternalLink href={ __( 'https://wordpress.com/support/wordpress-editor/' ) }>
```

Comment:
This should not be a translation but using `localizeUrl`.

Reply:
Thanks, @akirk. I have updated this in ab132c368ab56f9e5a4291cbdc984722d1f196e4.